framework/widgets/ActiveField.php

```diff
@@ -760,19 +760,20 @@ public function radioList($items, $options = [])
      * @param string $class the widget class name.
      * @param array $config name-value pairs that will be used to initialize the widget.
      * @return $this the field object itself.
+     * @throws \Exception
```

Comment:
?

Reply:
PHPDoc comment doesn't contain all necessary @throws tag
```php
$this->parts['{input}'] = $class::widget($config);
```

Reply:
It is useless annotation - basically every method may throw `\Exception`, and without any description it does not give you any value. Such noisy annotations are the reason why I disabled this inspection completely.

Reply:
probably, but I will observe the accepted practice